index.js

```diff
@@ -38,8 +38,8 @@ class BunyanStream extends EventEmitter {
             , app: record.name
             , context: Object.assign({}, record)
             , index_meta: true
+            , hostname: record.hostname
```

Comment:
as shown in [`logdna/logdna-winston`](https://github.com/logdna/logdna-winston/blob/master/index.js), `hostname` can be given thru `options` - so, I recommend to add `this.hostname = options.hostname` to the `constructor` and change this line to `, hostname: this.hostname || record.hostname`

Reply:
hm... in that case this change is redundant.  You already can set a hostname on the logger level with its current state. The idea is, how I understand and how it work in nodejs, is to be able to set an optional hostname on the line level

Reply:
@vilyapilya, I agree with that but what if a customer wants to have all lines having the same `hostname` - `record.hostname` does always exist but a customer may give optional `hostname` thru the constructor to override all `hostname`s; I mean, if `options` contains `hostname`, that means a customer wants to see that as a `hostname` for all lines - if it doesn't contain, then by default, it's always going to be `record.hostname`

Reply:
in that case, he adds the hostname to the options object he passes

Reply:
Yup - now I see - thanks for clarification!